exercises/armstrong-numbers/canonical-data.json

```diff
@@ -0,0 +1,86 @@
+{
+  "exercise": "armstrong-number",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "0 is an Armstrong number",
+      "property": "armstrong",
+      "input": {
+        "number": 0
+      },
+      "expected": true
+    },
+    {
+      "description": "1 is an Armstrong number",
+      "property": "armstrong",
+      "input": {
+        "number": 1
+      },
+      "expected": true
+    },
+    {
+      "description": "9 is an Armstrong number",
+      "property": "armstrong",
+      "input": {
+        "number": 9
+      },
+      "expected": true
+    },
+    {
+      "description": "10 is not an Armstrong number",
+      "property": "armstrong",
+      "input": {
+        "number": 10
+      },
+      "expected": false
+    },
```

Comment:
Are there no 2 digit Armstrong numbers?

Reply:
According to Wikipedia https://en.wikipedia.org/wiki/Narcissistic_number and https://oeis.org/A005188 there is none